src/evaluators/interpolation/PHAL_DOFCellToSide_Def.hpp

```diff
@@ -117,24 +140,229 @@ postRegistrationSetup(typename Traits::SetupData d,
 
   val_side.dimensions(dims);
 
+  for (int i = 0; i < dims.size(); ++i)
+    dimsArray[i] = dims[i];
```

Comment:
Can you throw an error here if dims.size() is larger than 5?

Reply:
Sure, I'll get that added.